scripts/build-types-index.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+// @ts-ignore
+// Right now we create types only for components we converted to typescript and we wrote in published-converted-components.
+// After we will convert all the source files we will change the import to webpack/published-components
+const { publishedComponents } = require("../webpack/published-converted-components");
+
+function buildComponentsTypesIndexFile() {
+  const content = Object.entries(publishedComponents)
+    .map(([name, path]) => `export { default as ${name} } from "./types/${path}";`)
```

Comment:
Does it exist for all components? If not, this could be an issue.

Reply:
We do not use the published component original file until we end to migrate all components, we are using a file which called published-converted-components until the migration will be done

Reply:
So why should it be a problem?